lib/utils/getModulePath.js

```diff
@@ -2,18 +2,25 @@
 "use strict";
 
 const configurationError = require("./configurationError");
+const resolve = require("path").resolve;
 const resolveFrom = require("resolve-from");
 
-module.exports = function(
+const GLOBAL_NODE_MODULES = resolve(__dirname, "../../../../node_modules/");
```

Comment:
Not everyones _global_ `node_modules` folder will be located at `../../../../node_modules/`

Reply:
Umm... what case can you assume?

Reply:
I'm just casually dropping this here, do ignore if it's irrelevant! I see `resolve-from` package is used already, how about also using [`resolve-global`](https://github.com/sindresorhus/resolve-global)? It has a fairly different approach compared to [`global-modules`](https://github.com/jonschlinkert/global-modules), with Window's Appdata and yarn support.

Reply:
@byCedric Thanks! `resolve-global` seems a nice tool to me. 👍 

But it doesn't support a Homebrew installed Node.js on macOS, so we may need both ways *relative path* and `resolve-global`.

For example, `resolve-global` with Homebrew installed Node.js:
( [`global-dirs`](https://github.com/sindresorhus/global-dirs) is internally used in `resolve-global` )

```js
const globalDirs = require("global-dirs")
console.log(globalDirs.npm.packages)
//=> /usr/local/Cellar/node/10.11.0/lib/node_modules
```

Our solution may be the following:

```diff
diff --git a/lib/utils/getModulePath.js b/lib/utils/getModulePath.js
index f71434cf..c808fb45 100644
--- a/lib/utils/getModulePath.js
+++ b/lib/utils/getModulePath.js
@@ -4,6 +4,7 @@
 const configurationError = require("./configurationError");
 const resolve = require("path").resolve;
 const resolveFrom = require("resolve-from");
+const resolveGlobal = require("resolve-global");
 
 const GLOBAL_NODE_MODULES = resolve(__dirname, "../../../../node_modules/");
 
@@ -18,6 +19,9 @@ module.exports = function getModulePath(
   if (!path) {
     path = resolveFrom.silent(process.cwd(), lookup);
   }
+  if (!path) {
+    path = resolveGlobal.silent(lookup);
+  }
   if (!path) {
     path = resolveFrom.silent(GLOBAL_NODE_MODULES, lookup);
   }
```

Reply:
Now I found [`global-modules`](https://github.com/jonschlinkert/global-modules). It uses internally [`global-prefix`](https://github.com/jonschlinkert/global-prefix) and it's search-path algorithm seems more safe to me.

This may make the relative-path approach unnecessary, so I'll try it. 💪 

Please give me any feedback if you have any time. 🙏

Reply:
Now [`global-modules`](https://www.npmjs.com/package/global-modules) has **191** dependents!

![image](https://user-images.githubusercontent.com/473530/46873639-0fb65000-ce72-11e8-8d69-622f9aabe6c4.png)

Reply:
I pushed 5c1397d and rebased the `master` branch.